src/olympia/editors/tests/test_forms.py

```diff
@@ -101,3 +94,41 @@ def test_no_app(self):
         # responses.
         assert len(choices) == 1
         assert self.cr_addon.response in choices[0]
+
+    def test_comments_and_action_required_by_default(self):
+        form = self.get_form()
+        assert not form.is_bound
+        form = self.get_form(data={})
+        assert form.is_bound
+        assert not form.is_valid()
+        assert form.errors == {
+            'action': [u'This field is required.'],
+            'comments': [u'This field is required.']
+        }
+
+        # Alter the action to make it require versions regardless of what it
+        # actually is, what we want to test is the form behaviour.
```

Comment:
I'm not following this comment - versions isn't being touched, comments being required is.